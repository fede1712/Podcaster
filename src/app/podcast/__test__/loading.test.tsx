import React from "react";
import { render, screen } from "@testing-library/react";
import Loading from "../loading";

test("Renders loading correctly", () => {
  render(<Loading />);

  const loadingElement = screen.queryByTestId("loading");
  expect(loadingElement).not.toBeNull();

  const { asFragment } = render(<Loading />);

  expect(asFragment()).toMatchSnapshot();
});
