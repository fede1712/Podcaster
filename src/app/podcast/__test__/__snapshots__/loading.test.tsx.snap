// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders loading correctly 1`] = `
<DocumentFragment>
  <span
    class="absolute top-10 right-20"
    data-testid="loading"
  >
    <span
      class="animate-ping absolute inline-flex h-full w-full rounded-full bg-sky-400 opacity-75"
    />
    <span
      class="relative inline-flex rounded-full h-3 w-3 bg-sky-500"
    />
  </span>
</DocumentFragment>
`;
